src/components/AddOrEditGroup/index.js

```diff
@@ -36,7 +36,8 @@ export default class EditGroupName extends PureComponent {
           this.handleLoading(true);
           addGroup({
             ...parameters,
-            ...vals
+            ...vals,
+            noModels: true
```

Comment:
This design is easy to forget.

Reply:
Otherwise, just use HandleError function ？